pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,213 @@
+  </tr>  
+  <tr>
+    <td><code>label</code></td>
+    <td>
+      The label that will be displayed in the pipeline visualisation in Buildkite (name of the group in the UI). Supports emoji.<br>
+      <em>Example:</em> <code>"\:hammer\: Tests" will be rendered as "🔨: Tests"</code><br>
+    </td>
+  </tr>
+  <tr>
+    <td><code>skip</code></td>
+    <td>
+      Whether to skip this group or not. Passing a string provides a reason for skipping this command. Passing an empty string is equivalent to <code>false</code>.
+      Note: Skipped steps will be hidden in the pipeline view by default, but can be made visible by toggling the 'Skipped jobs' icon.<br>
+      <em>Example:</em> <code>true</code><br>
+      <em>Example:</em> <code>false</code><br>
+      <em>Example:</em> <code>"My reason"</code>
+    </td>
+  </tr>
+</table>
+
+## Parallel groups  
+
+If you have two group steps next to each other, they run in parallel. For example:  
+
+```yml
+# 1.sh and 3.sh will start at the same time.
+# 2.sh will start when 1.sh finishes, and 4.sh will start
+# when 3.sh finishes.
+steps:
+  - group: "first"
+    steps:
+      - command: '1.sh'
+      - wait
+      - command: '2.sh'
+  - group: "second"
+    steps:
+      - command: '3.sh'
+      - wait
+      - command: '4.sh'
+```
+{: codeblock-file="parallel-groups.yml"}  
+
+Running jobs in parallel has some limitations:  
+
+<!-- vale off --> 
+* Parallel groups will be displayed ungrouped if the build's jobs are truncated because Buildkite doesn't currently store or calculate any information about the number of jobs in a non-parallel group.
+* If a parallel step exists within a group, parallel jobs are treated as regular jobs within a step group - so you can't have parallel groups within step groups. So, for example, a `group` that contains two `steps` each with `parallel: 4` will display eight jobs in it, with no visual indication that those eight jobs are two parallel steps.  
+
+<!-- vale on -->
+
+* If a parallel job group is within a named group, the groups are handled as though the parallel group isn't there.  
+* It's impossible to have a parallel job with only some of the jobs within a group, as they're all created on the same YAML step entry.  
+
+## Using wait steps in job groups
+ 
+You can have [wait steps](/docs/pipelines/wait-step) in a group. Such steps operate independently of other groups. For example, both groups will operate independently here, meaning "d" won't wait on "a" to finish:  
+
+```yml
+steps:
+  - group: "111"
+    depends_on: "tests"
+    steps:
+      - command: "a.sh"
+      - wait
+      - command: "b.sh"
+
+  - group: "222"
+    depends_on: "tests"
+    id: "toast"
+    steps:
+      - command: "c.sh"
+      - wait
+      - command: "d.sh"
+
+  - command: "yay.sh"
+    depends_on: "toast"
+```
+{: codeblock-file="pipeline-with-wait-steps.yml"}  
+
+## Group merging  
+
+Different groups that have the same name will be merged together in the Buildkite UI. 
```

Comment:
But do they run as separate groups and display as one? Or are they treated as one group?